src/index.tsx

```diff
@@ -1,13 +1,3 @@
-import * as ReactDOM from "react-dom";
-import App from "./App";
 import AddToCart from "./AddToCart";
-import React, { useRef } from "react";
 
-export const AddToCartComponent = AddToCart;
-ReactDOM.render(
-  <div>
-    <h1>Hello, Welcome to React and TypeScript</h1>
-    <App />
-  </div>,
-  document.getElementById("root"),
-);
+export default AddToCart
```

Comment:
if we do like this, we can't test that component in this project itself.

Reply:
ok sir , i try to do in another way

Reply:
this is working fine... export is for exporting at default namespace for the external npm package installed projects
ReactDOM.render is for testing within this project itself.